tools/gn

```diff
@@ -261,11 +261,6 @@ def to_gn_args(args):
 
   gn_args['full_dart_sdk'] = args.full_dart_sdk
 
-  if args.enable_unittests:
```

Comment:
We should retain this so that we can disable building the unit tests during an LTO build.

Reply:
Done.